packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/editing.css

```diff
@@ -0,0 +1,15 @@
+.root {
+    display: grid;
+    grid-template-rows: min-content 1fr;
+    row-gap: 1em;
```

Comment:
I think we should be using (r)elative em

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2280/commits/0115799684e2a042d57572f44b59328237c0298f.